src/main/java/org/folio/service/inventory/InventoryItemRequestService.java

```diff
@@ -0,0 +1,82 @@
+package org.folio.service.inventory;
+
+import static org.folio.service.inventory.InventoryUtils.INVENTORY_LOOKUP_ENDPOINTS;
+import static org.folio.service.inventory.InventoryUtils.REQUESTS;
+import static org.folio.service.inventory.util.RequestFields.DESTINATION_KEY;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.okapi.common.GenericCompositeFuture;
+import org.folio.rest.core.RestClient;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.core.models.RequestEntry;
+import org.folio.service.CirculationRequestsRetriever;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+public class InventoryItemRequestService {
+
+  private static final Logger logger = LogManager.getLogger(InventoryItemRequestService.class);
+
+  private static final String CANCEL_MESSAGE = "cancelRequest:: Cancelling Request with id='{}'";
+  private static final String MOVE_MESSAGE = "transferRequest:: Moving Request with id='{}' to item with id='{}'";
+  private static final String FILTER_ITEMS_MESSAGE = "getItemsWithActiveRequests:: Filtering itemIds with active requests: {}";
```

Comment:
these three messages used only once, so I propose to move back to logs

Reply:
Moved them, thanks